app/views/refills/_navigation.html.erb

```diff
@@ -21,6 +21,12 @@
           <%= link_to repositories_path do %>
             <%= t('.open_source') %>
           <%- end %>
+
+          <% if current_user.is_admin? %>
+            <%= link_to "/admin" do %>
```

Comment:
I haven't found `admin_path` related in the routes file. Please let me know if there is a better way of doing it.

Reply:
Use `admin_root_path`.

Reply:
@ramos-ph fix this please so we can merge this PR